addons/Rhombus/Rhombus.lua

```diff
@@ -558,7 +561,7 @@ function format_response(n,p,bool)
 end
 
 windower.register_event('keyboard', function(dik, down, flags, blocked)
-    if dik == 42 and not (bit.band(flags,32) == 32) then
+    if dik == 42 and flags < 32 then
```

Comment:
I agree that this is brittle, we make no promises about the stability of the unused flag bits.